base/src/com/google/idea/blaze/base/qsync/QuerySyncProjectData.java

```diff
@@ -102,7 +102,8 @@ public Collection<ProjectTarget> getReverseDeps(Path sourcePath) {
   public ImmutableList<TargetInfo> targets() {
     if (blazeProject.isPresent()) {
       return blazeProject.get().getTargetMap().values().stream()
-          .map(TargetInfo::fromBuildTarget)
+          .map(TargetInfo::tryFromBuildTarget)
+          .flatMap(Optional::stream)
```

Comment:
maybe do this without `Optional` and just `filter(Objects::nonNull)`?

Reply:
I like Optionals, but yes probably unnecessary heap pressure.